### ISSUE TITLE
Tablet: Problem Cards: Overview (Tablet > Dashboard)

### ISSUE THREAD

**Issue by sidmundhra** [benetech/MathShare#1459]
This task defines the overview of the problem cards on the individual set view of the dashboard. 

## Full Mockup
<img width="576" alt="Mockup of individual set view. The universal nav bar is on the top, followed by a back arrow and text All Sets on the left and Share Worklink button on the far right. Underneath the All sets arrow is the profile image followed by the problem set title Solve for X on the left, and a choose discipline dropdown followed by a three dot dropdown on the right. Underneath the set title, there's text Created at followed by the date the set was created and another text Last Updated at that displays the date it was last updated at. Both dates are in MMM DD YYYY format. Beneath the Set Title section are two problem cards, rounded rectangles with pastel backgrounds. Each card has the number in a small box in the top left, the prompt and equation in the middle of the card. Both cards have a black checkmark in a green circle also in the top right.." src="https://user-images.githubusercontent.com/12637929/96143638-46942600-0ed1-11eb-95a6-dd8e654d83f6.png">

The above mockup shows the issue in context of the individual set view.

### Notable changes from mockups: 
- We are not using classes or other metadata descriptors as shown in the mockup ("Algebra 101"), so this piece should be left out. 
- The numbers should increase sequentially, so the second problem card in this mockup would be "02" instead of "01". 
- The problem prompt should be in bold, whereas the problem equation should be in the bottom text (the first problem set card in the mockups is a clearer example than the second) 
- The problem cards should not have a three dot dropdown on the cards themselves in this tablet version (associated with problem set creation) 

## Layout 

The individual sets page of the dashboard should contain a card for each problem in the set. The cards should be rounded rectangles in a row from the top and continues to next row if there are more than 3 cards. Users should be able to scroll through these until they reach the last problem (progressive loading five problem cards at a time). 

Problem cards should be assigned a random color for their problem card background, which persists with that problem (e.g. if a problem is light blue, it will stay light blue after changes are saved to the problem set on the desktop version, and it will not be affected by new sets being added). 

These random colors should be one of the colors (light green, light orange, off-white) defined in the style guide in Invision and shown in the mockups above. 

## Reference
Style Guide + Official Mockups: https://projects.invisionapp.com/share/FKYLJNJGRMB#/screens/430773140

Tablet Mockups: https://projects.invisionapp.com/share/FKYLJNJGRMB#/screens/430773140?browse

Fonts: https://www.dropbox.com/sh/c12m6ufweyds20d/AAAdXNPOt8e52DkSAr9pd-a-a/Fonts?dl=0

## Acceptance criteria: 
Given that I'm a user on the Mathshare dashboard, when I click into a set, I should be able to scroll through all my problems in that set. The colors of each problem that I've created or worked on should stay a consistent color for me so I can associate the problem with that particular color over time. When I click on a problem set card, I should be brought to the problem solve view for that problem.

**Comment by alexrcabral** [comment]
Just a note that this is affected by #1458, in that the color assignment is not expected to persist

**Comment by johnhbenetech** [comment]
@alexrcabral One question that came up. In the mockups we see the submit and share buttons for tablet. Are these in scope for this task, i.e. do you want us to hook up that functionality and if so is there a mockup for that share modal?

Second followup, share/submit dont seem to have been in the mobile mockups (unless i missed them) so do we also want to add that functionality to the mobile view somehow?

**Comment by alexrcabral** [comment]
@johnhbenetech The submit and share buttons for tablet are not in scope for this task

**Comment by johnhbenetech** [comment]
@alexrcabral would you like us to omit the UI components from the implementation as well? or should we put the Ui in but not hook up?

**Comment by alexrcabral** [comment]
@johnhbenetech yes -- please omit the UI components for "submit" and "share" on this task

**Comment by abaranowska1** [comment]
**Tested with success**

As a user when I click into a set, I am able to scroll through all my problems in that set. The colors of each problem that I've created or worked on stays a consistent color for me so I can associate the problem with that particular color over time. When I click on a problem set card, I am brought to the problem solve view for that problem.

![Selection_528](https://user-images.githubusercontent.com/61964176/101176648-0f2b3580-3647-11eb-8fbb-97eb04295313.png)
![Selection_529](https://user-images.githubusercontent.com/61964176/101176641-0e929f00-3647-11eb-8e48-cefb933cc49e.png)
![Selection_530](https://user-images.githubusercontent.com/61964176/101176636-0d617200-3647-11eb-9a2f-1f06d5274dc5.png)

**Verified on:**
- Google Chrome responsive view 641x1007
- Ubuntu 20.04
- Safari Browser
- iPad Mini 2019, iOS v12 768x1024
